p2p/src/lib.rs

```diff
@@ -40,6 +41,10 @@ use io::{BufRead, Write};
 #[doc(inline)]
 pub use self::address::Address;
 
+#[rustfmt::skip]
+#[doc(inline)]
+pub use network_ext::NetworkExt;
+
```

Comment:
In line with the rest of the repo this could be:
```rust

#[rustfmt::skip]
#[doc(inline)]
pub use self::{address::Address, network_ext::NetworkExt};
```